confgenerator/config.go

```diff
@@ -208,8 +209,8 @@ func (l *Logging) Validate(platform string) error {
 	if err := validateComponentIds(l.Processors, subagent, "processor"); err != nil {
 		return err
 	}
-	if err := validateComponentIds(l.Exporters, subagent, "exporter"); err != nil {
-		return err
+	if len(l.Exporters) > 0 {
+		log.Print(`The "metrics.exporters" field is deprecated and will be ignored. Please remove it from your configuration.`)
```

Comment:
`The "metrics.exporters" field is deprecated as it's no longer needed, and will be ignored. This does not change any functionality. Please remove it from your configuration.`? Anything that can help assure users `nothing is wrong, please ignore and chill. No need to send a customer support case` is good with me.

Reply:
Updated. Tweaked the grammar slightly on your suggestion.